gen/gen.c

```diff
@@ -2941,10 +2952,44 @@ rfc2544_test(unsigned int n)
 		statistics_clear();
 		transmit_set(1, 1);
 
-		state = RFC2544_MEASURING0;
+		state = opt_rfc2544_warming_duration > 0 ? RFC2544_WARMING0 : RFC2544_MEASURING0;
```

Comment:
If `opt_rfc2544_warming_duration` is within `1-60`,  is this conditional expression unnecessary?

Reply:
Yes, I fixed this code in [b2bb884](https://github.com/iij/ipgen/pull/20/commits/b2bb884992ec4e947cd69f9969082422f197e2f4)